lute/static/js/dict-tabs.js

```diff
@@ -0,0 +1,445 @@
+"use strict";
+
+function createDictTabs(num = 0) {
+  // TERM_DICTS.push("https://de.thefreedictionary.com/###");
+  // TERM_DICTS.push("*https://glosbe.com/de/en/###");
+  // TERM_DICTS.push("*https://en.langenscheidt.com/german-english/###");
+  // TERM_DICTS.push("*https://en.pons.com/translate/german-english/###");
+  // TERM_DICTS.push("*https://www.collinsdictionary.com/dictionary/german-english/###");
+  // TERM_DICTS.push("*https://dict.tu-chemnitz.de/deutsch-englisch/###.html");
+  // TERM_DICTS.push("*https://www.translate.ru/%D0%BF%D0%B5%D1%80%D0%B5%D0%B2%D0%BE%D0%B4/%D0%BD%D0%B5%D0%BC%D0%B5%D1%86%D0%BA%D0%B8%D0%B9-%D0%B0%D0%BD%D0%B3%D0%BB%D0%B8%D0%B9%D1%81%D0%BA%D0%B8%D0%B9/###");
+
+  if (TERM_DICTS.length <= 0) return;
+
+  let sliceIndex;
+  let columnCount;
+
+  if (num < 0) num = 0;
+  if (num == 1) num = 2;
```

Comment:
This is unclear.  Why 2 tabs, if only 1 tab?  Shouldn't < 0 tabs throw an error?

Reply:
Unresolving, I just find this confusing.  num is an argument that's passed, I don't follow why it's changed to 2 if it's 1.